PowerShell/JumpCloud Commands Gallery/commands.json

```diff
@@ -97,6 +97,13 @@
     "link": "https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20CrowdStrike%20Falcon%20Agent.md",
     "description": "This command will download and install the CrowdStrike Falcon Agent to the device if it isn't already installed.\n\nIn order to use this command, follow the instructions from the [Installing the CrowdStrike Falcon Agent KB](https://support.jumpcloud.com/s/article/Installing-the-Crowdstrike-Falcon-Agent)\n\nSpecifically for this command:\n\n1. Download the CrowdStrike Falcon Agent installer and host it at a URL that your devices can access. Set this URL to the \"DownloadURL\" of this script.\n2. Extend the command timeout to a value that makes sense in your environment. The suggested command timeout for an environment with average network speeds on devices with average computing power is 10 minutes. Note that the command may timeout with a 124 error code in the command result window if not extended, but the script will continue to run.\n3. It is recommended to set the Launch Event for the command to \u201cRun As Repeating\u201d with an interval of one hour."
   },
+  {
+    "name": "Mac - Install Remote Assist | v1.0 JCCG",
```

Comment:
Are these additions intentional? All the remote assist stuff below too.

Reply:
this was an automated commit.  It looks like there is an automated workflow that updates this commands file [here](https://github.com/TheJumpCloud/support/blob/master/.circleci/workflows.yml#L316)

Reply:
We aren't ready to publish these commands just yet

Reply:
This isn't used just yet, so I think we're good to go.